package.json

```diff
@@ -245,6 +246,161 @@
       }
     ],
     "debuggers": [
+      {
```

Comment:
I'm thinking that as we start to add even more debuggers, this list is going to start to get fairly unmanageable, and since there's a lot of duplication anyway, I wonder if it makes sense to dynamically compile the final package.json instead of cramming everything in here. (don't have to change anything for this PR, just a thought).

Reply:
Yea that's what I'm thinking. Rob was hesitant on it before but editing this thing at this point is becoming a PITA. After you get your build PR in (so I don't step on your toes) I'll look at this if I have a few minutes.